deploy/cron.js

```diff
@@ -2,20 +2,19 @@ if (process.env.NODE_ENV !== 'production') {
   require('dotenv').config()
 }
 
+if (process.env.NEW_RELIC_APP_NAME) {
```

Comment:
loading new relic before winston, because 1. winston was having issues and 2. new relic's agent was complaining about not being loaded first (not being loaded before winston)